packages/berlin/src/pages/Cycle.tsx

```diff
@@ -58,7 +58,7 @@ function Cycle() {
   const [endAt, setEndAt] = useState<string | null>(null);
   const [localUserVotes, setLocalUserVotes] = useState<LocalUserVotes>([]);
   const [sorting, setSorting] = useState<{ column: string; order: Order }>({
-    column: 'pluralityScore',
+    column: 'numOfVotes',
```

Comment:
i dont like how this ordering affects my voting without it actually saving. Is there a way to order this only after saving?